platform/android/build.gradle.template

```diff
@@ -35,6 +35,9 @@ android {
 	packagingOptions {
 		exclude 'META-INF/LICENSE'
 		exclude 'META-INF/NOTICE'
+		exclude 'res/drawable/icon.png'
+		exclude 'res/layout/downloading_expansion.xml'
+		exclude 'res/layout/status_bar_ongoing_event_progress_bar.xml'
```

Comment:
They are actually included. Gradle made two of each those files in different location, and then complain about "duplication" when exporting. By doing exlude as suggested by Gradle itself, the problem resolved. And of course, those three files are still in their respective folders.